core/slice.go

```diff
@@ -1399,7 +1399,6 @@ func (sl *Slice) combinePendingHeader(header *types.WorkObject, slPendingHeader
 		combinedPendingHeader.Header().SetReceiptHash(header.ReceiptHash())
 		combinedPendingHeader.Header().SetEVMRoot(header.EVMRoot())
 		combinedPendingHeader.Header().SetUTXORoot(header.UTXORoot())
-		combinedPendingHeader.Header().SetEtxSetRoot(header.EtxSetRoot())
```

Comment:
why did you remove the etxsetroot from the combine method? we need this right? for async updates?